metrics_sdk/lib/opentelemetry/sdk/metrics/state/asynchronous_metric_stream.rb

```diff
@@ -49,6 +49,9 @@ def initialize(
           def collect(start_time, end_time)
             invoke_callback(@timeout, @attributes)
 
+            is_monotonic = @aggregation.respond_to?(:monotonic?) ? @aggregation.monotonic? : nil
+            aggregation_temporality = @aggregation.respond_to?(:aggregation_temporality) ? @aggregation.aggregation_temporality : nil
```

Comment:
This doesn't need to be addressed as part of this PR, but I just realized our tests for the metric streams are lacking. They seem to be hit through other tests with SimpleCov. I think we should consider adding class-specific tests in addition.

Reply:
Sure, I created the [issue](https://github.com/open-telemetry/opentelemetry-ruby/issues/1890)